c87019958e30
*** Found local files:
***   * /opt/airflow/logs/dag_id=Productos_ETL/run_id=manual__2023-06-18T05:37:18.845856+00:00/task_id=Conexion_Creacion_Insercion/attempt=2.log
[2023-06-18T05:43:05.691+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=Productos_ETL/run_id=manual__2023-06-18T05:37:18.845856+00:00/task_id=Conexion_Creacion_Insercion permission to 509
[2023-06-18T05:43:05.946+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=Productos_ETL/run_id=manual__2023-06-18T05:37:18.845856+00:00/task_id=Conexion_Creacion_Insercion permission to 509
[2023-06-18T05:43:08.183+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Productos_ETL.Conexion_Creacion_Insercion manual__2023-06-18T05:37:18.845856+00:00 [queued]>
[2023-06-18T05:43:08.262+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Productos_ETL.Conexion_Creacion_Insercion manual__2023-06-18T05:37:18.845856+00:00 [queued]>
[2023-06-18T05:43:08.293+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 3
[2023-06-18T05:43:08.492+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): Conexion_Creacion_Insercion> on 2023-06-18 05:37:18.845856+00:00
[2023-06-18T05:43:08.554+0000] {standard_task_runner.py:57} INFO - Started process 4198 to run task
[2023-06-18T05:43:08.597+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Productos_ETL', 'Conexion_Creacion_Insercion', 'manual__2023-06-18T05:37:18.845856+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/comercio.py', '--cfg-path', '/tmp/tmpmolhpjui']
[2023-06-18T05:43:08.612+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask Conexion_Creacion_Insercion
[2023-06-18T05:43:09.000+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=Productos_ETL/run_id=manual__2023-06-18T05:37:18.845856+00:00/task_id=Conexion_Creacion_Insercion permission to 509
[2023-06-18T05:43:09.049+0000] {task_command.py:410} INFO - Running <TaskInstance: Productos_ETL.Conexion_Creacion_Insercion manual__2023-06-18T05:37:18.845856+00:00 [running]> on host c87019958e30
[2023-06-18T05:43:09.816+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='DanielMendieta' AIRFLOW_CTX_DAG_ID='Productos_ETL' AIRFLOW_CTX_TASK_ID='Conexion_Creacion_Insercion' AIRFLOW_CTX_EXECUTION_DATE='2023-06-18T05:37:18.845856+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-18T05:37:18.845856+00:00'
[2023-06-18T05:43:11.999+0000] {logging_mixin.py:149} INFO - Conexión Exitosa.
[2023-06-18T05:43:12.627+0000] {logging_mixin.py:149} INFO - Tabla Creada.
[2023-06-18T05:43:19.885+0000] {logging_mixin.py:149} INFO - Datos Ingresados.
[2023-06-18T05:43:19.894+0000] {logging_mixin.py:149} INFO - Conexión Terminada.
[2023-06-18T05:43:19.908+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-06-18T05:43:20.037+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=Productos_ETL, task_id=Conexion_Creacion_Insercion, execution_date=20230618T053718, start_date=20230618T054308, end_date=20230618T054320
[2023-06-18T05:43:20.341+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-18T05:43:20.640+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
